homeassistant/config_entries.py

```diff
@@ -2397,6 +2397,7 @@ def async_update_reload_and_abort(
         data: Mapping[str, Any] | UndefinedType = UNDEFINED,
         options: Mapping[str, Any] | UndefinedType = UNDEFINED,
         reason: str = "reauth_successful",
+        always_reload: bool = True,
```

Comment:
I'm having mental problem with this naming.

The helper is `async_update_reload_and_abort`, but there  is an `always_reload`, to the `reload_and_abort` isn't always reload?

This is super confusing.

Reply:
Maybe my other suggestion is better?

https://github.com/home-assistant/core/pull/116026#discussion_r1576677836

`reload_if_unchanged`

Reply:
Yes, or even maybe `reload_even_if_entry_is_unchanged` (even more descriptive).